cartridges/int_adyen_SFRA/cartridge/forms/default/creditCard.xml

```diff
@@ -5,6 +5,9 @@
     <field formid="cardType" label="profile.cardtype" type="string" mandatory="false" binding="cardType"
            max-length="16" missing-error="error.message.required" />
 
+    <field formid="cardOwner" label="label.input.creditcard.owner" type="string" mandatory="false" binding="cardOwner"
+           max-length="16" missing-error="error.message.required" />
```

Comment:
is 16 max length enough?

Reply:
Maximum is 22. Will update it.

Reply:
Why do we need this as it will be rendered by the card component?